gallery.markdown

```diff
@@ -54,3 +54,39 @@ order: 1
   }
   </script>
 </figure>
+<figure itemscope="" itemtype="https://schema.org/Painting">
+  <img src="https://github.com/naomimakkelie-nl/www2.naomimakkelie.nl/assets/62353999/47ccca17-7c11-4788-879b-e90da6921c5e" alt="Battlefield on Valentine’s Day" itemprop="image">
+
+  <figcaption>
+    <pre itemprop="description" style="white-space: pre-line;">
+        This painting captures the juxtaposition of conflict and romance, embodying the chaos and passion of love. Set against the backdrop of a battlefield, it explores the themes of love, loss, and the enduring spirit of Valentine's Day amidst turmoil.
+    </pre>
+
+    <meta itemprop="width" content="70cm">
+    <meta itemprop="height" content="50cm">
+    <meta itemprop="artMedium" content="Watercolour, ink, pastelchalk on paper">
+    <meta itemprop="dateCreated" content="2022">
+    <meta itemprop="creator" content="Naomi Mitsuko Makkelie">
+    <meta itemprop="artworkSurface" content="Paper">
+    <meta itemprop="name" content="Battlefield on Valentine’s Day">
+  </figcaption>
+
+  <script type="application/ld+json">
+  {
+    "@context": "https://schema.org",
+    "@type": "Painting",
+    "name": "Battlefield on Valentine’s Day",
+    "description": "This painting captures the juxtaposition of conflict and romance, embodying the chaos and passion of love. Set against the backdrop of a battlefield, it explores the themes of love, loss, and the enduring spirit of Valentine's Day amidst turmoil.",
+    "width": "70cm",
+    "height": "50cm",
+    "artMedium": "Watercolour, ink, pastelchalk on paper",
+    "dateCreated": "2022",
+    "creator": {
+      "@type": "Person",
+      "name": "Naomi Mitsuko Makkelie"
+    },
+    "artworkSurface": "Paper",
+    "image": "https://github.com/naomimakkelie-nl/www2.naomimakkelie.nl/assets/62353999/47ccca17-7c11-4788-879b-e90da6921c5e"
```

Comment:
Instead of using the image that was uploaded to GitHub, save the image to the `/assets/img` directory inside the repositor and use the URL `https://www2.naomimakkelie.nl/assets/img/<filename>` to refer to the image.

Reply:
Sweep please make sure you add the image file to the repository in the assets/img directory and update the URL everywhere (in the JSON-LD as well as the img element)